composer.json

```diff
@@ -41,8 +41,8 @@
         "illuminate/routing": "5.5.*",
         "illuminate/validation": "5.5.*",
         "illuminate/view": "5.5.*",
-        "swiftmailer/swiftmailer": "~5.1",
-        "symfony/config": "2.7.*",
+        "swiftmailer/swiftmailer": "~6.0",
```

Comment:
Please don't include this change in this branch, add another commit rolling it back to the previous versions; that is a separate concern.